Update SIG_NAME_sandbox_stage.md (#275)
ossf/tac: process/templates/SIG_NAME_sandbox_stage.md

  * Link to relevant documentation.

### List SIG Lead(s)

> The WG must have a minimum of 1 Lead

Reviewer: Interesting point. Practically speaking though, someone always leads the calls, right? Maybe we can allow for an "Acting Lead"?

> The WG must have a minimum of 1 Lead

Reviewer 2: Yeah, what I'm trying to differentiate is if it's a requirement or not. There certainly can be an acting lead (or maybe the SIG already decided on a lead!) but I don't think it should be a requirement for the Sandbox stage.

>   * "name, affiliation, GitHUb ID"
> 

Reviewer 2: ```suggestion
```

Same as above.

### List of interested individuals
The SIG have a minimum of 3 members with 2 different organizational affiliations.
  * "name, affiliation, GitHub ID"

### Governing Body
SIGs may report to an existing OpenSSF Working Group or directly to the TAC as their governing body. The SIG commits to providing the governing body quarterly updates on progress.
  * "name of governing body (WG or TAC) for this SIG"

> ### Project References

Reviewer 2: ```suggestion
### SIG References
```

> The SIG should provide a list of existing resources with links to the repository, and if available, website, a roadmap, contributing guide, demos and walkthroughs, and any other material to showcase the existing breadth, maturity, and direction of the project.

Reviewer 2: ```suggestion
The SIG should provide a list of existing resources with links to the repository, and if available, website, a roadmap, demos and walkthroughs, and any other material to showcase the existing breadth, maturity, and direction of the SIG.
```

  * Link to relevant documentation.

### List of interested individuals

> The SIG have a minimum of two maintainers with different organizational affiliations.

Reviewer: Do we really have "maintainers" for SIGs?

Reviewer 3: +1. The SIG lifecycle doc uses the term "members", which would work fine here.

> The SIG have a minimum of two maintainers with different organizational affiliations.

Reviewer 2: ```suggestion
The SIG have a minimum of 3 members with 2 different organizational affiliations.
```
From https://github.com/ossf/tac/blob/main/process/sig-lifecycle.md?plain=1#L15

  * "name, affiliation, GitHub ID"

> ### Existing TI alignment

Reviewer 2: ```suggestion
### Governing Body
```

> The SIG should be aligned with an existing OpenSSF Working Group, or Project (TI).  The SIG commits to providing the hosting TI quarterly updates on progress.

Reviewer 2: ```suggestion
SIGs may report to an existing OpenSSF Working Group or directly to the TAC as their governing body. The SIG commits to providing the governing body quarterly updates on progress.
```

>   * "name of TI that will host this SIG"

Reviewer 2: ```suggestion
  * "name of governing body (WG or TAC) for this SIG"
```

### Project References
The SIG should provide a list of existing resources with links to the repository, and if available, website, a roadmap, contributing guide, demos and walkthroughs, and any other material to showcase the existing breadth, maturity, and direction of the project.
| Reference           | URL |
|---------------------|-----|
| Repo                |     |
| Meeting Agenda      |     |
| OSSF Calendar Entry |     |
| Website             |     |
| Contributing guide  |     |
| Security.md         |     |

> | Roadmap             |     |

Reviewer 4: ```suggestion
| Roadmap             |     |
| code-of-conduct.md             |     |
```
